### PR TITLE
Delete branch on PR merge

### PR DESCRIPTION
**Pull request by amdtech**
More for discussion than anything, haven't had a chance to test this yet.

Tagging @Drewzar for review.

### CODE REVIEW

**Comment by Drewzar** [comment]
I for one agree with this. If they need it back, they should be reverting the PR. :+1:

**Comment by amdtech** [comment]
Right... well, there's a "restore" button in the PR too, so it's not like it's gone forever :smile:

File: scripts/github.coffee [hunk 1]
```diff
@@ -63,3 +63,11 @@ module.exports = (robot) ->
   robot.on 'github:pullRequests:merge', (msg, options) ->
     github.pullRequests.merge options, (err, merge) =>
       handle_error(msg, "Issue merging pull request", err)
+
+      unless options.branch == "master" || err?
```
Comment (Drewzar) [inline scripts/github.coffee]: can you even delete master?

Comment (amdtech) [inline scripts/github.coffee]: __shrugs__ better safe than sorry?

**Comment by Drewzar** [comment]
Should we bother with this, or save it for when we move prod deploys to DOT?

**Comment by amdtech** [comment]
We should probably do this soonish, but I'd like to wait until jarvis' prod deploys are verified working in slack.

**Comment by Drewzar** [comment]
:+1:

**Comment by Drewzar** [comment]
:grapes:

**Pull request merged**